deploy/crds/operator.ibm.com_v1alpha1_metaoperatorcatalog_cr.yaml

```diff
@@ -20,3 +20,12 @@ spec:
     sourceNamespace: openshift-marketplace
     targetNamespaces:
       - etcd-operator
+  - name: ibm-cert-manager-operator
+    namespace: ibm-cs-certmanager
+    channel: <unsure>
+    packageName: ibm-cert-manager-operator
+    sourceName: <unsure>
+    sourceNamespace: <unsure>
+    targetNamespaces:
```

Comment:
@horis233 @chenzhiwei I think we will not need `targetNamespace` as we are planning to listen to all namespaces.

@Crystal-Chun we also need to add a `description` field here to clarify some info for your operator, like dependency etc.

Reply:
Thanks for the review @gyliu513! I've removed `targetNamespaces` and added a `description` field.

Two questions about `description`:
1. What is the format of it? 
2. What types of dependencies do we need to add? Cert-manager doesn't depend on other common services.

Reply:
> What is the format of it?

A string is good enough

> What types of dependencies do we need to add? Cert-manager doesn't depend on other common services.

OK, then adding some simple description for your service is good enough.

Reply:
Ok great, I'll remove the Kubernetes part of it. Thank you!